jobs/release-monitoring/discovery/github/Jenkinsfile

```diff
@@ -227,7 +231,11 @@ node {
         dir('installation') {
             checkoutGitRepo(installationGitUrl, installationGitRef, githubCredentialsID)
             releaseConfig = readYaml file: installationManifestFile
-            componentRelease = releaseConfig[manifestVar]
+            if (releaseTagVar != "") {
+              componentRelease = releaseConfig[releaseTagVar]
+            } else {
+              componentRelease = releaseConfig[productVersionVar]
+            }
```

Comment:
Bit unsure what this is for? Why do we use productVersionVar if releaseTagVar is empty? In what scenario would releaseTagVar be blank and why would we want to use productVersionVar instead in this case?

Reply:
The release tags are only ever used for operators. Operator release tag will be different from the product version they install. In cases like `che` or `3scale` the 'release tag' they use in Github is their actual product version.

Reply:
Do we still need this? Could we not set releaseTagVar to equal productVersionVar in the case where productVersionVar is the only thing we have?

```
def releaseTagVar =  params.manifestVar ? params.manifestVar : params.releaseTagVar ? params.releaseTagVar : params.productVersionVar
```